scripts/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -e
+
+# Cleanup
+rm -rf lib es
+
+echo 'Compile JS...'
+rollup -c scripts/rollup.config.js
+echo 'Done.'
+echo ''
+
+echo 'Build ES modules...'
+NODE_ENV=es babel src --out-dir es --ignore 'src/browser.js'
```

Comment:
I'd rather move that back to node scripts. We can't assume/test that'll work on all OS, especially windows.

Reply:
Sure, np. I shamelessly copied this from bootstrap-vue as that also has a build script by @pi0 :tm:

Reply:
Done, albeit under protest because bash is the best!